assets/dev/js/editor/controls/dimensions.js

```diff
@@ -92,27 +90,10 @@ ControlDimensionsItemView = ControlBaseUnitsItemView.extend( {
 	},
 
 	fillEmptyDimensions: function() {
-		var dimensions = this.getPossibleDimensions(),
```

Comment:
It causes a bug: Inside widget go to Advanced -> Padding, unlink the dimensions and change one of the fields. It won't work